modules/apps/transfer/types/events.go

```diff
@@ -18,4 +18,5 @@ const (
 	AttributeKeyAck            = "acknowledgement"
 	AttributeKeyAckError       = "error"
 	AttributeKeyTraceHash      = "trace_hash"
+	AttributeKeyMetadata       = "amount"
```

Comment:
```suggestion
	AttributeKeyMetadata       = "metadata"
```

Reply:
fixed

Reply:
Looks like maybe the fixing commit needs to be pushed?